packages/server/README.md

```diff
@@ -92,6 +92,12 @@ First, install Apollo Server, the JavaScript implementation of the core GraphQL
 npm install @apollo/server graphql express cors body-parser
 ```
 
+If using Typescript you may also need to install additional type declaration packages/modules to avoid common errors when importing the above packages (i.e. Could not find a declaration file for module '`module-name`'):
```

Comment:
```suggestion
If using Typescript you may also need to install additional type declaration packages as development dependencies to avoid common errors when importing the above packages (i.e. Could not find a declaration file for module '`cors`'):
```